src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+    private double height;
+
+    public IsoscelesTrapezoid(String color,
+                              double firstLeg,
+                              double secondLeg,
+                              double height) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        this.height = height;
+    }
+
+    public double getHeight() {
+        return this.height;
+    }
+
+    public void setHeight(double height) {
+        this.height = height;
+    }
+
+    public double getSecondLeg() {
+        return this.secondLeg;
+    }
+
+    public void setSecondLeg(double secondLeg) {
+        this.secondLeg = secondLeg;
+    }
+
+    public double getFirstLeg() {
+        return this.firstLeg;
+    }
+
+    public void setFirstLeg(double firstLeg) {
+        this.firstLeg = firstLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return (this.firstLeg + this.secondLeg) * this.height / 2;
```

Comment:
The calculation of the area is incorrect for an isosceles trapezoid. The formula should be the average of the two parallel sides (bases) multiplied by the height. Please correct the formula.